sed.py error: unknown function: I
